packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -112,7 +114,7 @@ export class BundlingDockerImage {
       ...options.user
         ? ['-u', options.user]
         : [],
-      ...flatten(volumes.map(v => ['-v', `${v.hostPath}:${v.containerPath}`])),
+      ...flatten(volumes.map(v => ['-v', `${v.hostPath}:${v.containerPath}:${v.consistency ? v.consistency : 'delegated'}`])),
```

Comment:
```suggestion
      ...flatten(volumes.map(v => ['-v', `${v.hostPath}:${v.containerPath}:${v.consistency ?? DockerMountConsistency.DELEGATED}`])),
```

Reply:
Changed to use the shorter version of the ternary operator.